src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public void getRandomBall() {
+        for (int i = 0; i < 3; i++) {
+            Ball ball = new Ball();
+            Random random = new Random();
+            ColorSupplier supplier = new ColorSupplier();
+            ball.setNumber(random.nextInt(100));
+            ball.setColor(supplier.getRandomColor().toString());
+            System.out.println("Ball color is: " + ball.getColor()
+                    + " ball number is: "
+                    + ball.getNumber());
+        }
+    }
```

Comment:
```suggestion
public class Lottery {
    private static final Random random = new Random();
    private static final ColorSupplier supplier = new ColorSupplier();
            
    public Ball getRandomBall() {
        Ball ball = new Ball();
        ball.setNumber(random.nextInt(100));
        ball.setColor(supplier.getRandomColor().toString());
        return ball;
    }
```
Lets make ColorSupplier and Random global - the same reason as above
This method should return just one ball, getRandomBall not balls
You can invoke this method as many times as you wish in main
This class should be responsible for creating just one ball